src/devices/machine/t10mmc.cpp

```diff
@@ -962,12 +972,7 @@ void t10mmc::ReadData( uint8_t *data, int dataLength )
 						buffer_offset = 0;
 						data_len = 2352;
 					}
-					else if (track_type == cdrom_file::CD_TRACK_MODE1)
-					{
-						buffer_offset = 0;
-						data_len = 2048;
-					}
-					else if (track_type == cdrom_file::CD_TRACK_MODE1_RAW)
+					else if (track_type == cdrom_file::CD_TRACK_MODE1 || track_type == cdrom_file::CD_TRACK_MODE1_RAW)
```

Comment:
Bug fix for a regression introduced in https://github.com/mamedev/mame/pull/11499 that I noticed while doing regression testing for this PR. I didn't take into account that I was taking advantage of the partial promotion to mode1_raw when reading `cdrom_file::CD_TRACK_MODE1` tracks, so user data section accidentally included the 16 byte header as user data. This lets `taiko6` boot again.